flink-python/pyflink/fn_execution/embedded/operations.py

```diff
@@ -45,13 +50,15 @@ def __init__(self,
                  output_data_converter: DataConverter,
                  runtime_context,
                  function_context,
+                 timer_context,
                  job_parameters):
         operations = (
             [OneInputOperation(
                 function_urn,
```

Comment:
This would make chained operations like (keyed, non-keyed, non-keyed) to produce a chain of three keyed operations.

Reply:
Yes. I will change these part logic.